.github/workflows/release-nightly.yml

```
@@ -100,3 +100,14 @@ jobs:
     secrets:
       GH_ORG: ${{ secrets.GH_ORG }}
       GH_TOKEN: ${{ secrets.GH_TOKEN }}
+
+  keycloak-quickstarts:
+    name: Keycloak QuickStarts
+    needs: [create-tags,keycloak]
```

I've been seeing this explicit dependency declaration before, is that intentional? Since `keycloak` already has a dependency on `create-tags` this job could also be defined as follows:

```suggestion
    needs: [keycloak]
```

I am neutral on this, WDYT?